quizgen/latex.py

```diff
@@ -19,24 +26,33 @@ def is_available():
 
     return True
 
-def compile(path):
-    bin_path = 'pdflatex'
-    if (_pdflatex_bin_path is not None):
-        bin_path = _pdflatex_bin_path
-
-    result = subprocess.run([bin_path, '-interaction=nonstopmode', os.path.basename(path)], cwd = os.path.dirname(path),
-        capture_output = True)
-
+def compile(path,use_docker=False):
+    if use_docker:
+        dir_path = os.path.dirname(os.path.abspath(path))
+        file_name = os.path.basename(path)
+        cmd = [
+            "docker", "run", "--rm",
+            "-v", f"{dir_path}:/workspace",
+            "edulinq-quizgen-tex",
+            "pdflatex", "-interaction=nonstopmode", file_name
```

Comment:
Options like `-interaction=nonstopmode` you can put in the CMD,
or if we use a script you can just include in the script.
Not everyone will always want this option, but since we are creating our own image we can do things how we want.